modules/misunderstood/src/views/full/index.tsx

```diff
@@ -307,6 +309,52 @@ export default class MisunderstoodMainView extends React.Component<Props, State>
     })
   }
 
+  clickImportData = () => {
+    this.importInput.current.click()
+  }
+
+  importData = evt => {
```

Comment:
leave TODOs + make tickets:
- replace with generic file upload component
- might want to swap endpoint from JSON to proper file upload / download